client/app/dbaas/metrics/token/metrics-token.html

```diff
@@ -10,95 +10,35 @@ <h3 class="oui-header_3" data-translate="metrics_token_title"></h3>
             data-ng-disabled="MetricsTokenCtrl.actions.loading"
             data-translate="metrics_token_new"
             ui-sref="dbaas.metrics.detail.token.add({ serviceName: MetricsTokenCtrl.serviceName })"></a>
-        <div class="pull-right">
+        <!-- <div class="pull-right">
```

Comment:
useless comment block

Reply:
Done. Reimplement filter following last update of the list-view